src/towncrier/newsfragments/501.feature

```diff
@@ -0,0 +1 @@
+Calling ``towncrier check`` without an existing configuration, will just show only an error message. In previous versions, a traceback was also generated.
```

Comment:
```suggestion
Calling ``towncrier check`` without an existing configuration, will just show only an error message. In previous versions, a traceback was generated instead of the error message.
```